### ISSUE TITLE
Error when running docker container (yara.go:59 runner.go:81)

### ISSUE THREAD

**Issue by cherez0ff** [deepfence/SecretScanner#131]
```
docker run -i --rm \
	-v /var/run/docker.sock:/var/run/docker.sock \
	quay.io/deepfenceio/deepfence_secret_scanner_ce:2.4.0 \
	-image-name alpine:latest
```

Got error:
```
ERRO[2024-10-08T09:03:52Z] yara.go:59 open : no such file or directory
ERRO[2024-10-08T09:03:52Z] runner.go:81 error in runOnce compiling yara rules: open : no such file or directory
```

Same for older version.

**Comment by ramanan-ravi** [comment]
Hello @cherez0ff, running secret scanner in cli mode is not supported in this version.
It will be added in the next major release.

**Comment by DavidTanner** [comment]
@ramanan-ravi that is confusing, because those are the directions given on the website.  What is the major version that will support the CLI?

**Comment by ramanan-ravi** [comment]
Hello @DavidTanner, we will let you know once it's available

**Comment by ramanan-ravi** [comment]
Hello @DavidTanner @cherez0ff, cli support is now available in v2.5.0
